### DISCUSSION TITLE
WorkLikeElon— Productivity Chrome Extension/ App For Software Engineers

### DISCUSSION THREAD

**Discussion by thecodearrow** [scaleracademy/open-source-projects#16] [Ideas]
**Motivation**

With remote work pretty much becoming the norm and having to do everything online with very little compartmentalization (thanks to the pandemic), "deep work" is becoming harder and harder to achieve every passing day. There are so many productivity tools out there and yet being productive and learning to use your time effectively is becoming so challenging on a day-day basis. 

Yet, learning to effectively use your time is very critical for one's mental health, daily happiness, and career over the long term.  The goal is to build the ultimate productivity tool for software engineers from feedback taken from members in our community.  

**Features**

- Customised Pomodoro (_25-5-15 may not work for everyone_)
- Daily Focus (_What's the one thing that needs to be completed today?_)
- Autoplay songs from YouTube URLs/ playlist. 
- Daily, weekly and monthly analytics. (_This needs to be done well, since tracking your progress and constant feedback is essential for one to stay motivated_)
- Blocking off social media or custom sites during the deep work session (_open to experimentation_).
- Motivational quotes (_can't say no to that_)
- Free of cost and zero ads. (_A lot of the other extensions or apps charge quite a lot!_ )
- Gamification would be cool too( _we can use a lot of ideas from Habitica_)
- Other features can be added/ removed, based on user feedback. 

I'm open to brainstorming more feature suggestions on making this a perfect productivity tool for software engineers and how we can build something completely different from what's out there.

**Comment by pri1311** [comment]
Love the name XD

I use a few of such tools and google calendar as well, so my suggestion would be to automatically add tasks/events from my google calendar to daily tasks section maybe. Cause having both in one would be great

**Comment by championswimmer** [comment]
Do you think this would be a Chrome extension or a standalone desktop app ?

**Reply by thecodearrow** [reply]
To start with, I think the MVP can be a chrome extension. 

If needed, I think we should be able to convert it to a desktop app too using React Native.
